src/frontend/screens/Game/GamePicture/index.tsx

```diff
@@ -6,10 +6,17 @@ import fallbackImage from 'frontend/assets/heroic_card.jpg'
 
 interface Props extends React.ImgHTMLAttributes<HTMLImageElement> {
   art_square: string
+  logo?: string | undefined
   store: string
 }
 
-function GamePicture({ art_square, store, className, ...props }: Props) {
+function GamePicture({
+  art_square,
+  logo = undefined,
```

Comment:
same here as the other places, I'd just call this `art_logo` and without the default `undefined`, makes naming more consistent with the other places and it's undefined if not defined already